proxyd/config.go

```diff
@@ -128,6 +136,8 @@ type BackendGroupConfig struct {
 	ConsensusHARedis             RedisConfig  `toml:"consensus_ha_redis"`
 
 	Fallbacks []string `toml:"fallbacks"`
+
+	MulticallBackends []string `toml:"multicalls"`
```

Comment:
didn't we dropped this?

once the bg is configured to multicall, all backends in the group should be multicall backends

Reply:
Correct, its dropped. This is unused and will be removed